src/Confluent.Kafka/Consumer.cs

```diff
@@ -27,701 +27,15 @@
 
 namespace Confluent.Kafka
 {
-    /// <summary>
```

Comment:
this class has been moved to it's own file and contains some straightforward modifications, I'll note those modifications in that file.